src/index.js

```diff
@@ -1,50 +1,141 @@
 import './style.css';
-import square from './assets/img/square.svg';
-import moreDots from './assets/img/more-vertical.svg';
-
-const lists = [
-  {
-    description: 'wash the dishes',
-    completed: false,
-    index: 0,
-  },
-  {
-    description: 'sweep the floor',
-    completed: false,
-    index: 1,
-  },
-  {
-    description: 'clean the room',
-    completed: false,
-    index: 2,
-  },
-];
+import './assets/img/square.svg';
+import './assets/img/more-vertical.svg';
+import './assets/img/bin.svg';
 
+// add
 const todoEl = document.querySelector('.todo-list');
 
-const addList = (desc) => {
-  const lists = document.createElement('div');
-  lists.classList = 'lists';
-  const checkBox = document.createElement('img');
-  checkBox.src = square;
-  checkBox.classList = 'check-box';
-  checkBox.alt = 'square';
+class Lists {
+  constructor() {
+    this.tdlists = JSON.parse(localStorage.getItem('tdlists')) || [];
+  }
 
-  const todoText = document.createElement('input');
-  todoText.classList = 'todo-text';
-  todoText.value = desc;
-  todoText.disabled = true;
+  delList(item, idx) {
+    this.tdlists = JSON.parse(localStorage.getItem('tdlists')) || [];
+    const tdIdx = this.tdlists[idx].index;
+    this.tdlists.splice(tdIdx, 1);
+    this.tdlists.forEach((tdlist, tdnum) => {
+      tdlist.index = tdnum;
+    });
 
-  const more = document.createElement('img');
-  more.src = moreDots;
-  more.classList = 'more';
-  more.alt = 'more-vertical';
+    localStorage.setItem('tdlists', JSON.stringify(this.tdlists));
+    this.showList();
+  }
 
-  lists.append(checkBox, todoText, more);
+  showList() {
+    todoEl.innerHTML = '';
+    this.tdlists.forEach((tdlist) => {
+      const lists = document.createElement('div');
+      lists.classList = 'lists';
+      const checkBox = document.createElement('img');
+      checkBox.src = './assets/img/square.svg';
+      checkBox.classList = 'check-box';
+      checkBox.alt = 'square';
```

Comment:
- [ ]  I think it would be better to create a new JavaScript file for the new functionality as having different files for different functionalities would be easier to locate specific code when editing or debugging. Additionally, it can make it easier to reuse code by separating it into different files. 😎😎